input/intro-notes/StructureDefinition-au-coverage-intro.md

```diff
@@ -2,3 +2,11 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When populating the type using the preferred ValueSet :
+    - *Medicare* coverage **SHOULD** be represented by sending the V3 Value SetActCoverageTypeCode code "PUBLICPOL"
```

Comment:
You've correctly implemented this line as written in the ticket. 
However, it's not correct. Should be 

- *Medicare* coverage **SHOULD** be represented by sending the v3.ActCode code "PUBLICPOL"

Reply:
I am unable to make changes to the resolution.

Reply:
I think this is an editorial problem for the resolution - I am ok with referring to the source Value Set - but "V3 Value SetActCoverageTypeCode" should be "HL7 v3 Value Set ActCoverageTypeCode" as per https://hl7.org/fhir/R4/v3/ActCoverageTypeCode/vs.html for all of these